src/cuga/backend/server/manage_routes/config_routes.py

```
@@ -0,0 +1,437 @@
+
+        _prev_hash = ""
+        _prev_adapt_hash = ""
+        _prev_gloss_hash = ""
+        try:
+            _prev_config, _ = await load_config(version=None, agent_id=agent_id)
+            if _prev_config:
+                _prev_hash = (_prev_config.get("knowledge") or {}).get("_vector_config_hash", "")
+                _prev_adapt_hash = (_prev_config.get("knowledge") or {}).get("_adaptation_hash", "")
+                _prev_gloss_hash = (_prev_config.get("knowledge") or {}).get("_glossary_hash", "")
+        except Exception:
+            pass
```

Ruff (0.15.20)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)